docs/source/index.rst

```diff
@@ -119,6 +119,26 @@ Furthermore, ``S3FileSystem.current()`` will return the most-recently created
 instance, so this method could be used in preference to the constructor in
 cases where the code must be agnostic of the credentials/config used.
 
+Self-hosted S3
+--------------
+
+To use S3Fs against your self hosted S3-compatible storage, like `MinIO`_ or
```

Comment:
Prefer lowercase `s3fs` (with backticks to get the code style)

Reply:
Updated :+1: